app/src/main/java/com/android/voyageur/model/user/UserViewModel.kt

```diff
@@ -258,8 +258,17 @@ open class UserViewModel(
    * @param onSuccess callback for the response
    */
   fun getUsersByIds(userIds: List<String>, onSuccess: (List<User>) -> Unit) {
+    _isLoading.value = true
```

Comment:
Nice catch, I forgot to set the `isLoading` value on this before

Reply:
Thanks! 👍